packages/bsp/common/etc/update-motd.d/10-armbian-header

```diff
@@ -92,6 +92,11 @@ fi
 # Display software vendor logo
 echo -e "\e[1;91m$(figlet -f small " $VENDOR")\e[0m";
 
+# Read RPI model from cpuinfo
+if [[ $BOARD == rpi4b ]]; then
+BOARD_NAME=$(grep "Model" /proc/cpuinfo | sed -E 's/^Rev //; s/ Rev [0-9.]+$//' | cut -d":" -f2 | xargs)
```

Comment:
Can we read this directly from `/proc/device-tree/model` instead?

From what I can tell that's where cpuinfo's model comes from anyway.

Reply:
Its the same info + would need additional computing:
grep: /proc/device-tree/model: binary file matches

Reply:
That's just the zero terminator at the end, right? `tr '\0' '\n'` should do the trick.

The grep and cut steps would be unnecessary then, so we win some we lose some :)

```suggestion
BOARD_NAME=$(cat /proc/device-tree/model | tr '\0' '\n' | sed -E 's/ Rev [0-9.]+$//')
```

essentially: `cat /proc/device-tree/model | tr '\0' '\n' | sed -E 's/ Rev [0-9.]+$//'`

(I also left out the first sed command, do RPI model strings ever start with "Rev "?)